cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/OperationsDashboardHttpHandler.java

```diff
@@ -103,9 +104,40 @@ public void readDashboardDetail(FullHttpRequest request, HttpResponder responder
       throw new BadRequestException("'namespace' cannot be empty, please provide at least one namespace.");
     }
     long endTimeSecs = startTimeSecs + durationTimeSecs;
+
+    Collection<RunRecordDetail> programsRunningAtStartTime;
```

Comment:
Declare it at where it is initialized (line 116)

Reply:
Updated.